types/shared.d.ts

```diff
@@ -0,0 +1,162 @@
+declare module 'libsession_util_nodejs' {
+  type Uint8ArrayFixedLength<T extends number> = {
+    buffer: Uint8Array;
+    length: T;
+  };
+  /**
+   * Allow a single type to be Nullable. i.e. string => string | null
+   */
+  type Nullable<T> = T | null;
```

Comment:
Might be good to export Nullable for use in session desktop? Same for AllFieldsNullable

Reply:
exported